packages/app-store/server.ts

```diff
@@ -81,24 +82,23 @@ export async function getLocationGroupedOptions(
   integrations.forEach((app) => {
     if (app.locationOption) {
       // All apps that are labeled as a locationOption are video apps. Extract the secondary category if available
-      let category =
+
+      let groupByCategory =
```

Comment:
Changed the variable name to `groupByCategory` to clearly convey the meaning of category here.